demaceo/binary
# Feature news

#### What does  this PR do?
- Makes successful API call to `getBreakingNews()`.
- Creates `Article` component for each individual object returned from the `breakingNews` array.
- Displays API data for _News_ and _Article_ components.
- Creates and adds functionality to `useLocalStorage()`.
- Adds `onDoubleClick` feature to each _Article_ component that opens a link to the article in a new tab.
- Refactors how the `date` in the _Header_ component is adjusted.
#### How should this be manually tested?
- Double click on one of the _Articles_ in the 'Home' page.
- Double click on either arrow in the _Header_ to see the `date` accurately change.
#### What are the relevant tickets?
Closes #8
